cmd/azure-devops/smoke-tests/smoke-tests.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"context"
+	"log"
+	"os"
+	"strconv"
+
+	"github.com/kyma-project/test-infra/pkg/azuredevops/pipelines"
+)
+
+func main() {
+	// Fetching environment variables for Azure DevOps settings
+	organizationURL := os.Getenv("ORGANIZATION_URL")
```

Comment:
Organization URL and project name are present in kaniko-build-config.yaml file. Consider adding logic to read them for that file. Similar like in an image-builder. I think the name should be changed to something not related to kaniko only.

Reply:
I am against this change. This solution is more flexible. 

https://github.tools.sap/kyma/module-manifests/pull/193/files#diff-4bf95a92d7e0d7d307e3030d0931cb346ecc69beb52bcf7a8c579d13d34d8a96R141